remo/base/static/base/css/app-fd4.less

```diff
@@ -732,8 +732,8 @@ label.required:after {
         text-align: left;
     }
     small.error {
-        width: 50%;
-        padding: 0;
+        width: 100%;
```

Comment:
can you please test this change with other errors that we show? For example on the profile page or a new event page?

Reply:
That rule involve only the featured-reps page and not the others and now with the new layout we need to change it.

Reply:
oh, yes, sorry!